rhel/Dockerfile

```diff
@@ -57,12 +57,7 @@ RUN curl -sSL https://packages.instana.io/Instana.gpg -o /tmp/Instana.gpg && \
     echo "256bd490a55a1939a4c9cd15c043404b79a86429ee04129c00d33dab8c0cf040 /usr/bin/runc" | sha256sum -c - > /dev/null && \
     chmod 755 /usr/bin/runc && \
     rm -rf /tmp/* /etc/yum.repos.d/Instana-Agent.repo /etc/yum.repos.d/docker.repo && \
-    microdnf clean all && \
-    curl https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /usr/bin/jq && \
```

Comment:
**question:** can we safely ignore jq & pip during installation?

Reply:
pip is noe in python3, jq already installed above.

Reply:
`python` + `python-setuptools` + `pip` (manually installed) got superseded by just `python3`, already including all the other necessary dependencies.
`jq` is installed a few lines above already, so this was obsolete.